src/utils/DockerUtil.js

```diff
@@ -212,6 +213,7 @@ export default {
   run (name, repository, tag) {
     tag = tag || 'latest';
     let imageName = repository + ':' + tag;
+    name = ContainerStore.generateName(name);
```

Comment:
I put it in DockerUtil.run because when a user opens Kitematic using a URL scheme it will create the name before Kitematic has finished loading preexisting containers so if a user had a hello-world and hello-world-2 container it will overwrite the hello-world container rather than name it hello-world-3.  Would it make sense to only generate the name there or do you have another suggestion on how to deal with that?